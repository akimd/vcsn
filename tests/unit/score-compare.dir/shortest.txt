  0     1     2     3
[31m[1m22.70[0m [31m[1m28.70[0m [32m[1m 0.65[0m [32m[1m 0.63[0m a.shortest(5)        # a = de_bruijn(9), 10x
[31m[1m 1.26[0m [31m[1m 0.93[0m [32m[1m 0.20[0m [32m[1m 0.19[0m a.shortest(5)        # a = std([a-e]?{600}), c = [a-e] -> Z
[34m  N/A[0m [34m  N/A[0m [31m[1m 0.71[0m [32m[1m 0.67[0m a.shortest(5000)     # a = de_bruijn(9), 10x
[34m  N/A[0m [34m  N/A[0m  0.28[0m  0.27[0m a.shortest(5000)     # a = derived_term((\e|x + a|\e)*), c = [a]?x[x]? -> Q, 10x
  0     1     2     3
  0. v2.0-0001-g6bfe026
  1. v2.0-0423-g841c067
  2. v2.0-1255-gafaf339
  3. v2.1-0010-gd55ef26
