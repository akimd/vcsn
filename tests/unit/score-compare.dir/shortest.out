[31m[1m22.70[0m [31m[1m28.70[0m [32m[1m 0.65[0m [32m[1m 0.63[0m a.shortest(5)        # a = de_bruijn(9), 10x
[31m[1m 1.26[0m [31m[1m 0.93[0m [32m[1m 0.20[0m [32m[1m 0.19[0m a.shortest(5)        # a = std([a-e]?{600}), c = [a-e] -> Z
  N/A   N/A [31m[1m 0.71[0m [32m[1m 0.67[0m a.shortest(5000)     # a = de_bruijn(9), 10x
  N/A   N/A  0.28[0m  0.27[0m a.shortest(5000)     # a = derived_term((\e|x + a|\e)*), c = [a]?x[x]? -> Q, 10x
